src/setup/intro.md

```diff
@@ -83,6 +83,10 @@ Before installing any component of OutSystems, make sure your hardware and softw
 
 * [Default Platform Server and database configurations](https://success.outsystems.com/Documentation/11/Setting_Up_OutSystems/Default_Platform_Server_and_database_configurations): to install and run OutSystems, you need permission to change files in the file system, run services, and make changes to the database.
 
+## On-Premises Pre-Installation Video Example
```

Comment:
These videos are more appropriate for training than the general overview provided in this article. They will be forward to the training team for this attention.